script.js

```diff
@@ -73,28 +70,39 @@ function init() {
         keyboardDiv.className = 'keys';
```

Comment:
I can't write in line before, but it is not good have document.body.append(...) a lot of time. Try to use only one time, use wrapper for all elements or DocumentFragment https://habr.com/ru/post/413287/